docs/org-units/getting-web-api.md

```diff
@@ -2,3 +2,99 @@
 title: Getting org units from DHIS2 Web API
 short_title: Getting from Web API
 ---
+This page shows practical ways to fetch organisation unit (OU) data from the DHIS2 Web API using `dhis2-python-client` library.
+
+In the **climate-tools** environment, `dhis2-python-client` library is already installed. You only need to come up with your DHIS2 credintials and server address to use this library. Below is a basic usage:
+
+```{code-cell} python
+from dhis2_client import DHIS2Client
+from dhis2_client.settings import ClientSettings
+
+# Client configuration
+cfg = ClientSettings(
+  base_url="http://localhost:8080",
+  username="admin",
+  password="district")
+
+client = DHIS2Client(settings=cfg)
+
+info = client.get_system_info()
```

Comment:
Suggest to remove empty line between 19 and 21 so they "belong together".
And then empty line before the comment "# Check if..." so that's a new section.

Reply:
done